src/SDKs/ContainerInstance/Management.ContainerInstance/Microsoft.Azure.Management.ContainerInstance.csproj

```diff
@@ -2,7 +2,7 @@
   <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
   <PropertyGroup>
     <Description>Microsoft Azure Container Instance Management Library</Description>
-    <VersionPrefix>1.0.0-preview</VersionPrefix>
+    <VersionPrefix>1.0.1-preview</VersionPrefix>
```

Comment:
@yolo3301 missing package release notes like below
https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/BatchAI/Management.BatchAI/Microsoft.Azure.Management.BatchAI.csproj#L10

Reply:
@shahabhijeet the spec PR was already in the description. Pasting here again: Azure/azure-sdk-for-python#1557

Reply:
@shahabhijeet Have been using the wrong PR. The right one is: https://github.com/Azure/azure-rest-api-specs/pull/1863  Also updated the one in the description.